package-lock.json

```diff
@@ -5249,16 +5255,24 @@
 					"requires": {
 						"lcid": "^1.0.0"
 					}
-				},
-				"yargs-parser": {
-					"version": "5.0.0-security.0",
-					"resolved": "https://registry.npmjs.org/yargs-parser/-/yargs-parser-5.0.0-security.0.tgz",
-					"integrity": "sha512-T69y4Ps64LNesYxeYGYPvfoMTt/7y1XtfpIslUeK4um+9Hu7hlGoRtaDLvdXb7+/tfq4opVa2HRY5xGip022rQ==",
-					"dev": true,
-					"requires": {
-						"camelcase": "^3.0.0",
-						"object.assign": "^4.1.0"
-					}
+				}
+			}
+		},
+		"yargs-parser": {
```

Comment:
why is this section changed?

Reply:
I don't exactly know, just running an `npm install` caused this, but it just looked like it did some reorganizing so I wasn't worried about it.

Reply:
It looks good. Thanks a lot for the great work in this PR!